md/share-a-project-on-github.md

```diff
@@ -1,11 +1,11 @@
-# How to share a repository on Github 
+# How to share a project on Github 
```

Comment:
I would not rename the file, allowing to switch from one version of Bonita to another, staying on the same 'page'

Reply:
is it a frequent use case for customers ?

Reply:
When on a wrong version, you click in the menu corresponding to your version, if the page has the same name, you stay on the page, which is convenient.
So yes.

Reply:
What about search and seo ? Does it indexes the article name ?

Reply:
Switching version and staying on the same page is a use case.
Keeping repository becomes an issue.
=> Switch to "project" and create a redirect when coming from the same page on previous versions?

Reply:
I suggest to keep the same file name.
When changing from BonitaBPM to Bonita, we agreed to keep the same file names, because of SEO and because of bookmarking. Why should we do different this time?

Reply:
It is the backup solution that we will adopt if the redirect is too much of a pain.
I'll let you determine the complexity.

Reply:
final decision is not to change the md name.